.pre-commit-config.yaml

```diff
@@ -2,12 +2,14 @@
 # See http://pre-commit.com/hooks.html for more hooks
 repos:
 - repo: git://github.com/antonbabenko/pre-commit-terraform
-  rev: v1.7.0
+  rev: v1.7.1
+  sha: 091f8b15d7b458e5a0aca642483deb2205e7db02
   hooks:
     - id: terraform_fmt
-    - id: terraform_docs
+    # - id: terraform_docs
```

Comment:
If you disable this hook it won't update documentation automatically. Not sure you want this.

Reply:
The `terraform_docs` hook should not fail - this is the most important part. Please help me to figure out why it fails because I want people to use hook for real in real projects. (I tested it on Mac OS X and it seems to have some issues for some people on other OS). Maybe you can open an issue there.

The primary feature of pre-commit hooks is that developers who push PR or maintainers like you and me get them installed once and forget. The code is automatically formatted, the documentation is updated once a change is being committed - as a result, documentation is along with the code in the same PR.

It also simplifies manual steps like you have to do now during release.

I don't have strong objections if you prefer to do it another way, absolutely :)